### ISSUE TITLE
PyDDNS with SSL

### ISSUE THREAD

**Issue by shimuldn** [olimpo88/PyDDNS#11]
I am trying to configure Let's Encrypt SSL.
I have very limited knowledge on docker so I am stuck.

Checking from [the internet ](https://medium.com/faun/setting-up-ssl-certificates-for-nginx-in-docker-environ-e7eec5ebb418) and other but not sure how to do it.

**Comment by olimpo88** [comment]
This project does not integrate Let's Encrypt, but you can do it with nginx, that guide should work for you only need add the volumen path in the docker-compose.yml

```
    volumes:
      - ./config/nginx_src:/src
      - ./config/nginx:/etc/nginx/conf.d
      - ./appdata/static:/static
     - ./data/certs:/etc/nginx/certs   <-- add this
```
Create a folder data/certs and put the certificate file there
Finally modify the nginx configuration file in config/nginx/mydjango.conf

Important  inside your docker the certificate is in this path **/etc/nginx/certs/**

---------------------------------------------------------


Another solution is to use haproxy + cerbot
This is useful because you can expose many systems with a single port

https://github.com/lonequantum/docker-haproxy-certbot

Basically you have to put a reverse proxy in front that serves port 443, then you can send those connections to port 80 of PyDDNS.

**Comment by DSorlov** [comment]
I have made a pull request (#13) that implements this from the start. It would assume SSL over not and offer a redirect.
So if you have not solved it already @shimuldn you could see the changes needed in that pull request. =)

**Comment by shimuldn** [comment]
> I have made a pull request (#13) that implements this from the start. It would assume SSL over not and offer a redirect.
> So if you have not solved it already @shimuldn you could see the changes needed in that pull request. =)

Thanks, will check.
